src/markdown-pages/docs/cli-command-reference/cli-command-reference-about.md

```diff
@@ -0,0 +1,39 @@
+---
+path: "/docs/cli-command-reference/cli-command-reference-about"
+title: "Galasa CLI commands"
+---
+
+Galasa CLI commands start with `galasactl`. Example commands are provided for running on Mac or Unix, and Windows Powershell. The Windows Powershell uses the backtick (`) for line continuation characters. If you are using Windows command-shell, the line continuation character is the caret (^). 
+
+You can view the Galasa CLI command syntax, including parameter descriptions, in the <a href=https://github.com/galasa-dev/cli/blob/main/docs/generated/galasactl.md target="_blank"> cli repository</a> in GitHub.
+
+
+## Getting help
```

Comment:
I still think it's weird talking about this before you get to the install instructions.

Reply:
have switched around